pkg/policy/report.go

```diff
@@ -41,71 +32,45 @@ func (pc *PolicyController) report(engineResponses []*response.EngineResponse, l
 }
 
 // forceReconciliation forces a background scan by adding all policies to the workqueue
-func (pc *PolicyController) forceReconciliation(reconcileCh <-chan bool, cleanupChangeRequest <-chan policyreport.ReconcileInfo, stopCh <-chan struct{}) {
+func (pc *PolicyController) forceReconciliation(reconcileCh <-chan bool, cleanupChangeRequest <-chan string, stopCh <-chan struct{}) {
```

Comment:
Same as above...what should be passed in the string.

Reply:
Sure

Reply:
fixed